testing/tests/DevExpress.ui.events/swipe.tests.js

```diff
@@ -387,6 +387,21 @@ QUnit.test('default behaviour on dxpointermove should be prevented to reduce use
     $element.trigger($.Event('dxpointerup', { pointerType: 'mouse', pointers: [] }));
 });
 
+QUnit.test('Event should not be prevented that avoid the [Intervation] error when event is not cancelable', function(assert) {
+    const $element = $('#element');
+
+    $element.on(swipeEvents.start, noop);
+
+    $element.trigger($.Event('dxpointerdown', { pointerType: 'mouse', pageX: 200, pageY: 200, pointers: [0] }));
+
+    const moveEvent = $.Event('dxpointermove', { cancelable: false, pointerType: 'mouse', pageX: 210, pageY: 200, pointers: [0] });
+    $element.trigger(moveEvent);
+
+    assert.strictEqual(moveEvent.isDefaultPrevented(), false, 'event is not prevented');
+
+    $element.trigger($.Event('dxpointerup', { pointerType: 'mouse', pointers: [] }));
```

Comment:
y do we need this `trigger`? Seems like the test works perfect event without this

Reply:
You are right, it is not require